website/content/en/docs/contribution-guidelines/release.md

```diff
@@ -214,15 +197,29 @@ To verify a release binary using the provided asc files see the [installation gu
 These steps describe how to conduct a release of the SDK, upgrading from `v1.2.0` to `v1.3.0`.
 Replace these versions with the current and new version you are releasing, respectively.
 
-For major and minor releases, `master` should be locked between steps 1 and 3 so that all commits will be either in the new release
+For major and minor releases, `master` should be locked between steps 3 and 6 so that all commits will be either in the new release
 or have a pre-release version, ex. `v1.2.0+git`. Otherwise commits might be built into a release that shouldn't be.
 For patch releases, ensure all required bugs are [cherry-picked](#cherry-picking), then the release branch `v1.3.x` should be locked down.
 
+### 1. Netlify configuration
+
 **Important:** ensure a release branch-to-subdomain mapping exists in the SDK's Netlify configuration _prior to creating a release_,
 ex. `v1.3.x` to `https://v1-3-x.sdk.operatorframework.io`. You can ping SDK [approvers][doc-owners] to ensure a
 [release branch](#release-branches) is created prior to the release and that this mapping is created.
 
-### 1. Create a PR for release version, CHANGELOG.md, and migration guide updates
+### 2. Create release branch for Netlify
+
+The release branch must be created before the release occurs to appease the Netlify website configuration demons.
+You can do so by running the following before proceeding with the release, assuming the upstream SDK is the `upstream` remote repo:
+
+```sh
+$ git checkout master
+$ git pull
+$ git checkout -b v1.3.x
+$ git push -u upstream v1.3.x
```

Comment:
```suggestion
git checkout master
git pull
git checkout -b v1.3.x
git push -u upstream v1.3.x
```

Reply:
The reason; we are trying to follow this approach since it is helpful for we copy and paste the commands.